src/server/game/Entities/Player/Anticheat.cpp

```diff
@@ -0,0 +1,229 @@
+#include "Player.h"
+#include "Language.h"
+
+#define DEFAULT_PLAYER_BOUNDING_RADIUS      0.388999998569489f     // player size, also currently used (correctly?) for any non Unit world objects
+
+void Player::SetUnderACKmount()
+{
+    m_mountTimer = 3000;
+    m_ACKmounted = true;
+}
+
+void Player::SetRootACKUpd(uint32 delay)
+{
+    m_rootUpdTimer = 1500 + delay;
+    m_rootUpd = true;
+}
+
+bool Player::CheckOnFlyHack()
+{
+    if (IsCanFlybyServer())
+        return true;
+
+    if (ToUnit()->IsFalling() || IsFalling())
+        return true;
+
+    if (m_mover != this)
+        return true;
+
+    if (GetMapId() == 616 || GetMapId() == 649)
+        return true;
```

Comment:
like this you can disable some mapid for some trouble instances

Reply:
It would be nice if the map list could be configurable via `worldserver.conf`. So it would be easy to quickly `.reload conf` and try to switch on/off the AntiCheat while trying a specific instance. Testing would benefit from that

Reply:
@kvipka excellent!